jest.base.js

```diff
@@ -4,10 +4,7 @@ module.exports = {
     },
     collectCoverage: true,
     collectCoverageFrom: ['src/**/*.ts'],
-    coverageReporters: [
-        'text', 
-        ['lcov', { projectRoot: '../../' }]
-    ],
```

Comment:
Did automatic formatting change these lines? Ideally, pure reformatting should be left out of PRs

Reply:
Formatting the file when adding `snapshotFormat` below caused this.